net_util/src/tap.rs

```diff
@@ -383,6 +383,11 @@ impl Tap {
     pub fn get_if_name(&self) -> Vec<u8> {
         self.if_name.clone()
     }
+
+    #[cfg(fuzzing)]
+    pub fn new_for_fuzzing(tap_file: File, if_name: Vec<u8>) -> Self {
```

Comment:
Given it's already behind the build guard `fuzzing`, I'd call this function `new()`.

Reply:
I tried that, but unfortunately that did not work, because this custom constructor has a different signature (e.g. taking different arguments) from the existing `Tap::new()`.

Reply:
I see.